LayoutTests/animations/3d/full-rotation-animation.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <style>
+    .box {
+      height: 80px;
+      width: 80px;
+      background-color: red;
+    }
+
+    .test {
+      height: 80px;
+      width: 80px;
+      perspective: 250px;
+      border: 1px solid black;
+      background: green;
+    }
+
+     @keyframes animation {
+         0% { transform: rotateY(0); }
+         100% { transform: rotateY(-1turn); }
+    }
+  </style>
+</head>
+<body>
+
+<div class="test">
+  <div class="inner box">
+  </div>
+</div>
+
+<script>
+    let target = document.querySelector('.box');
+    getComputedStyle(target).getPropertyValue("transform");
+    target.style.animationName = 'animation';
+    target.style.animationTimingFunction = 'linear';
+    target.style.animationDuration = '10000000000s';
+    target.style.animationDelay = '-2500000000s';
```

Comment:
Why are those required vs. just setting them in the `<style>` element above?

Reply:
Good point. I've moved these into the `<style>` element.